src/announcement/AnnouncementController.ts

```diff
@@ -23,43 +23,43 @@ interface NotificationMap {
 /**
  * A map of notification ids to StateNotification objects
  */
-export interface StateNotificationMap {
+export interface StateAnnouncementMap {
   [id: number]: StateNotification;
 }
 
 /**
- * NotitificationConfig will hold the active notifications
+ * AnnouncementConfig will hold the active announcements
  */
-export interface NotificationConfig extends BaseConfig {
+export interface AnnouncementConfig extends BaseConfig {
   allNotifications: NotificationMap;
 }
 
 /**
- * Notification state will hold all the seen and unseen notifications
+ * Announcement state will hold all the seen and unseen announcements
  * that are still active
  */
-export interface NotificationState extends BaseState {
-  notifications: StateNotificationMap;
+export interface AnnouncementState extends BaseState {
+  notifications: StateAnnouncementMap;
```

Comment:
This is part of the public API, and is still called `notifications`.

This is especially problematic because of how the extension flattens controller state. Leaving this here would not just be confusing, but it would preclude the use of the top-level key `notifications` in other controllers.

Reply:
Yeah, I was wondering about that. I can change it but it'll mean that we will have to merge the migration PR https://github.com/MetaMask/metamask-extension/pull/14389 with the controllers bump.

Reply:
At this point I might as well rename everything. Done in https://github.com/MetaMask/controllers/pull/697/commits/faa4ae062d5a04f102380ad3bd6c32a9ee157a16